src/main/ContinuousIntegrationServer.java

```diff
@@ -1,19 +1,28 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import javax.servlet.ServletException;
- 
+
+import java.io.File;
 import java.io.IOException;
  
 import org.eclipse.jetty.server.Server;
 import org.eclipse.jetty.server.Request;
 import org.eclipse.jetty.server.handler.AbstractHandler;
+import org.eclipse.jgit.api.CloneCommand;
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.api.PullCommand;
+import org.eclipse.jgit.api.PullResult;
+import org.eclipse.jgit.merge.MergeStrategy;
```

Comment:
just do"import org.eclipse.jgit.*"?

Reply:
Why ?

Reply:
less code, but doesn't matter.

Reply:
I think writting it like this makes it more clear on what is actually used. We do not do any useless imports to.